announcements/templates/announcements/create_announcement.html

```diff
@@ -0,0 +1,65 @@
+{% extends "web/base.html" %}
+{% load i18n %}
+
+{% block body %}
+    <div class="ui container">
+        <h1>{% block header_text %}{% trans "New Announcement" %}{% endblock %}</h1>
+        <form class="ui form" method="POST" action="">
+            {% csrf_token %}
+            <div class="ui two fields">
+                <div class="ui field">
+                    <label>{{ form.classification.label }}</label>
+                    {{ form.classification }}
+                </div>
+                <div class="ui field">
+                    <label>{{ form.link.label }}
+                        <span data-content="{% trans "An optional link to an information page" %}">
```

Comment:
Maybe use the `help_text` Django field attribute for these?

Reply:
Absolutely! I have never seen that field before, TIL.